scalameter-core/src/main/scala/org/scalameter/picklers/package.scala

```diff
@@ -22,6 +22,9 @@ package object picklers {
     implicit val stringListPickler = StringListPickler
     implicit val longSeqPickler = LongSeqPickler
     implicit val dateOptionPickler = DateOptionPickler
+
+    // complex
```

Comment:
Could you expand this comment?

If you are starting a new section for more complex picklers, please use the syntax:

    /* More complex picklers */

And delimit it with one empty line before and after.